handlers/deploy.go

```diff
@@ -118,16 +116,19 @@ func MakeDeployHandler(functionNamespace string, clientset *kubernetes.Clientset
 
 func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets map[string]*apiv1.Secret, config *DeployHandlerConfig) (*v1beta1.Deployment, error) {
 	envVars := buildEnvVars(&request)
-	path := filepath.Join(os.TempDir(), ".lock")
 	probe := &apiv1.Probe{
 		Handler: apiv1.Handler{
-			Exec: &apiv1.ExecAction{
-				Command: []string{"cat", path},
+			HTTPGet: &apiv1.HTTPGetAction{
+				Path: "/healthz",
+				Port: intstr.IntOrString{
+					Type:   intstr.Int,
+					IntVal: int32(watchdogPort),
+				},
 			},
 		},
-		InitialDelaySeconds: 3,
+		InitialDelaySeconds: 0,
 		TimeoutSeconds:      1,
-		PeriodSeconds:       10,
+		PeriodSeconds:       1,
```

Comment:
This should also be a configuration item with a default of the previous value for compatibility. When used in dispatch you'd just set your values via the helm chart

Reply:
Are you sure you want the default to be the previous value(s)?  These values will have far more positive effect than negative, and shouldn't break anything existing